src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
@pathikrit, yes. This is repo is targeting 2.13. 

In order to release sooner you should target https://github.com/scala/scala, but I don't think your contribution will be included in 2.12 lifecycle due to binary compatibility considerations. If your collection is included in, lets say, 2.12.3, it's clearly incompatible with 2.12.0, as it didn't include this collection. 

Based on this, I think that the earliest you collection can be included is next binary incompatible release, which is 2.13. And this means that you should target this repo and base your contributions on this strawman.

Another alternative would be for you to release it as an artifact which is a separate artifact on maven.